tests/Translators/PlannedEvents/test_pe_tim_translator.py

```diff
@@ -0,0 +1,98 @@
+from unittest.mock import patch
+import Translators.PlannedEvents.pe_tim_translator as pe_tim_translator
+from Translators.Shared.itis_codes import ItisCodes
+
+def test_calculate_direction_eastbound():
+    coordinates = [(0, 0), (1, 0)]
+    assert pe_tim_translator.calculate_direction(coordinates) == "I"
+
+def test_calculate_direction_westbound():
+    coordinates = [(1, 0), (0, 0)]
+    assert pe_tim_translator.calculate_direction(coordinates) == "D"
+
+def test_calculate_direction_northbound():
+    coordinates = [(0, 0), (0, 1)]
+    assert pe_tim_translator.calculate_direction(coordinates) == "I"
+
+def test_calculate_direction_southbound():
+    coordinates = [(0, 1), (0, 0)]
+    assert pe_tim_translator.calculate_direction(coordinates) == "D"
+
+@patch('Translators.PlannedEvents.pe_tim_translator.query_db', return_value=[])
+def test_translate(mock_query_db):
+    pe_geojson = {
+        "features": [
+            {
+                "geometry": {
+                    "coordinates": [(0, 0), (1, 0), (2, 0)]
+                },
+                "properties": {
+                    "id": "test_id",
+                    "additionalImpacts": ["Impacts Both Directions"],
+                    "routeName": "test_route",
+                    "type": "Closed for the Season"
+                }
+            }
+        ]
+    }
+    expected_output = {
+        "timRcList": [
+            {
+                "clientId": "test-name",
+                "direction": "B",
+                "route": "test-route",
+                "roadCode": "test-name",
+                "itisCodes":[ItisCodes.CLOSED_FOR_SEASON.value],
+                "geometry": [{
+                    "latitude": 0,
+                    "longitude": 0
+                }, {
+                    "latitude": 0,
+                    "longitude": 1
+                }, {
+                    "latitude": 0,
+                    "longitude": 2
+                }],
+                "advisory": []
+            }
+        ]
+    }
+    assert pe_tim_translator.translate(pe_geojson) == expected_output
+
+@patch('Translators.PlannedEvents.pe_tim_translator.query_db', return_value=[])
+def test_translate(mock_query_db):
```

Comment:
suggestion: rename so we differentiate from the method above and qualify what we're testing in the name

Reply:
Updated